lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -132,10 +137,19 @@
 def install_flat(module, binary, remote, name, method):
     """Add a new flatpak."""
     global result
-    if name.startswith('http://') or name.startswith('https://'):
-        command = "{0} install --{1} -y {2}".format(binary, method, name)
+    apps = " ".join(name)
+    use_remote = True
+
+    for app in name:
+
+        if app.startswith('http://') or app.startswith('https://'):
+            use_remote = False
```

Comment:
What if some apps in the list `name` start with `https://` and some don't, with the `use_remote=False` case those that don't start with `https://` to fail installation? If so, there should be some book keeping done here to handle that.

Reply:
Thank you for bringing this up! I have rebased my patch and added an update to deal with a mix of URLs and non-URLs (just names) in the list.